npm/cypress-schematic/src/schematics/ng-generate/e2e/index.ts

```diff
@@ -49,7 +49,7 @@ export default function (options: Schema): Rule {
     }
 
     if (options.path === undefined) {
-      options.path = `${project.root}/cypress/integration`
+      options.path = `${project.root}/cypress/e2e`
```

Comment:
Did we talk to the DX team about how this schematic will be updated during release? Because this is a public change that requires a release of the schematic in master. It should probably be updated to work with Cypress < 10.0 and Cypress 10.0+ differently?

Reply:
I'll ping DX. My understanding is Cypress 10 and this change will go out at the same time, but that (might? need to learn more about schematics) cause a problem you outlined - users on Cypress 9 will get the new `cy.js` pattern when generating a new spec using the newly released Cypress 10 branch when it's merged.

Reply:
Edit: DX is aware of this: https://cypressio.slack.com/archives/CNM0FKJFJ/p1641517910008700